narwhal/node/src/restarter.rs

```diff
@@ -85,6 +85,10 @@ impl NodeRestarter {
             handles.extend(primary_handles);
             handles.extend(worker_handles);
 
+            // give some time to the node to bootstrap before we are ready to receive
+            // another reconfiguration message
+            tokio::time::sleep(tokio::time::Duration::from_secs(2)).await;
```

Comment:
why is this needed?

Reply:
When we bootstrap the node we don't have strong signal whether the node has been fully bootstrapped or there are still components that try to start. That give us some slack in. case we immediately try to reconfigure epochs back to back - mostly an issue observed in tests. In the upcoming iterations will try to introduce readiness signals when components bootstrap